Codecademy/client-modules
# feat(monaco): add monaco package

### Overview


Initial setup of shared monaco package. 

Multiple places on our site (LE, Codebytes, Sandbox) require a code editor. The goal of this shared package is to make a code editor with a standardized configuration available in multiple applications. Read more in this [RFC](https://www.notion.so/codecademy/Monaco-editor-as-a-shared-package-1f4484db165b4abc8394c3556451ef6a)



### PR Checklist

- [x] Related to JIRA ticket: [Reach-942](https://codecademy.atlassian.net/browse/REACH-942)

### Testing with codebytes

1. Visit the codebytes-editor [here](https://tayra.codecademy.com/codebyte-editor)
2. Select a language and run some code 
3. Confirm code runs and resizing of the editor works
